RELEASE_NOTES/3.47.md

```diff
@@ -0,0 +1,24 @@
+# App
+
+* ***New `roles` field for `shopify.web.toml`.*** Unify web processes, having more of the same environment variables in all cases. Multiple `roles` can now be passed instead of a single `type`. Also, `background` webs are allowed with neither frontend nor backend properties. More info: https://shopify.dev/docs/apps/tools/cli/structure.
+* ***Allow Theme App Extensions to exclude files from deploying via `.shopifyignore` file***.
+* ***Function extensions are now supported by 'shopify app dev'***. They are built and updated in extension drafts on dev startup, and on file change.
```

Comment:
This isn't supported yet I believe, it's going to be supported soon though. You need the beta flag now.